src/__tests__/CryptoCurrency.test.js

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import CryptoCurrency from '../components/CryptoCurrency';
+import '@testing-library/jest-dom/extend-expect';
+
+jest.mock('react-redux');
```

Comment:
- [ ] Good job adding tests for the `CryptoCurrency` page, kindly do so for the `Details` page as well as this is a requirement to test several components of our app.